src/popup/pages/SignTransaction.vue

```diff
@@ -51,8 +58,24 @@ export default defineComponent({
           return;
         }
 
+        const replaceCallerTx = async (tx: Encoded.Transaction) => {
+          const unpackedTx = unpackTx(tx);
+          if (unpackedTx.tag === Tag.ContractCallTx || unpackedTx.tag === Tag.ContractCreateTx) {
```

Comment:
```suggestion
          if ([Tag.ContractCallTx, Tag.ContractCreateTx].includes(unpackedTx.tag)) {
```

Reply:
nice suggestion, will add it

Reply:
unfortunately then the compiler doesn't know the existence of `.nonce` as well